Tests/Integration/MSFT_xFirewall.Integration.Tests.ps1

```diff
@@ -37,50 +70,82 @@ if ($executionPolicy -ne 'Unrestricted')
     Set-ExecutionPolicy -ExecutionPolicy Unrestricted -Force
     $rollbackExecution = $true
 }
+#endregion
+
+# Using try/finally to always cleanup even if something awful happens.
+try
+{
 
-try {
-    # Load in the DSC Configuration
-    . $PSScriptRoot\Firewall.ps1
+####################################################################################################
 
-    Describe 'xFirewall_Integration' {
+    <#
+      This file exists so we can load the test file without necessarily having xNetworking in
+      the $env:PSModulePath. Otherwise PowerShell will throw an error when reading the Pester File.
+    #>
+    $ConfigFile = Join-Path -Path $PSScriptRoot -ChildPath "$DSCResourceName.Config.ps1"
```

Comment:
Should probably be a lowercase C to match the file name: $DSCResourceName.config.ps1